build.sbt

```diff
@@ -360,3 +377,31 @@ lazy val benchmarks =
       }.taskValue
     )
     .enablePlugins(ScalaNativePlugin)
+
+commands ++= Seq(
+  Command.command("cleanAll") { state =>
+    "clean" ::
+      "cleanCache" ::
+        "cleanLocal" ::
+          state
+  },
+  Command.command("publishLocalAll") { state =>
+    "project testInterface" ::
+      "+ publishLocal" ::
+        "project /" ::
+          "reload" ::
+            "nscplugin/publishLocal" ::
+              "nativelib/publishLocal" ::
+                "publishLocal" ::
+                  state
+  },
+  Command.command("testAll") { state =>
+    "tools/test" ::
+      "sandbox/run" ::
+        "demoNative/run" ::
+          "tests/run" ::
+            "benchmarks/run" ::
+              "scripted" ::
+                state
+  }
+)
```

Comment:
@sjrd @Duhemm is it the nicest way to aggregate tasks? Can't we just redefine `test`, `publishLocal` and `clean` on the root project?